jobs/datalake-data-export/src/services/read-model-queries.service.ts

```diff
@@ -13,21 +13,51 @@ export class ReadModelQueriesService {
 
   public async getEServices(): Promise<Array<ExportedEService>> {
     return this.readModel.eservices
-      .find()
+      .find({
+        $or: [
+          {
+            'data.descriptors.1': { $exists: true },
+          },
+          {
+            'data.descriptors': {
+              $size: 1,
+              $elemMatch: {
+                state: { $in: ['Published', 'Archived', 'Suspended', 'Deprecated'] satisfies DescriptorState[] },
+              },
+            },
+          },
+        ],
+      })
```

Comment:
An e-service is considered "Draft" if has no descriptor, or has one with "Draft" state.
We need to filter them out from the result.